pkgs/default.nix

```diff
@@ -3639,24 +3640,31 @@ self: {
           description = "Convert strings into hexadecimal and back";
           license = stdenv.lib.licenses.bsd3;
         }) {};
-      hinotify = callPackage ({ async, base, containers, directory, mkDerivation, stdenv, unix }:
+      hfsevents = callPackage ({ Cocoa, CoreServices, base, bytestring, cereal, mkDerivation, mtl, stdenv, text }:
       mkDerivation {
-          pname = "hinotify";
-          version = "0.3.9";
-          sha256 = "16fzql0s34my9k1ib4rdjf9fhhijkmmbrvi148f865m51160wj7j";
+          pname = "hfsevents";
```

Comment:
@domenkozar I might need to pick up your brain once again on this 😞 

I'm at the stage where this _should_ work, but when I ran `pkgs/generate.sh` even with the latest master, it's generating something which I fear is not correct -- why is it pulling `hfsevents`, which is darwin only, instead of `hinotify`? I have the dreadful suspect this is because I'm running `pkgs/generate.sh` from darwin, but I thought that your [patch](https://github.com/input-output-hk/cardano-sl/pull/1314) fixed such behaviour? 🤔